tools/main-transpile-wrapper.js

```diff
@@ -0,0 +1,8 @@
+// This file exists because it is the file in the tool that is not automatically
+// transpiled by Babel.
+
+require('source-map-support').install();
+require('babel/register')(require('./babel-config.js')); // #RemoveInProd this line is removed in isopack.js
```

Comment:
https://github.com/babel/babel/blob/master/src/babel/api/register/cache.js

In your home directory, but there's an env var to override it, and we should probably make the top level shell script do that when running from a checkout to put it inside the checkout instead (gitignored)